src/coreaudiokit.cs

```diff
@@ -60,6 +74,81 @@ interface AUAudioUnitViewControllerExtensions {
 		[Export ("selectViewConfiguration:")]
 		void SelectViewConfiguration (AUAudioUnitViewConfiguration viewConfiguration);
 	}
+
+	[NoiOS]
+	[Mac (10,13, onlyOn64: true)]
+	[Protocol]
+	interface AUCustomViewPersistentData {
+
+		[Abstract]
+		[NullAllowed, Export ("customViewPersistentData", ArgumentSemantic.Assign)]
+		NSDictionary<NSString, NSObject> CustomViewPersistentData { get; set; }
+	}
+
+	[NoiOS]
+	[Mac (10,13, onlyOn64: true)]
+	[DisableDefaultCtor] // Crashes
+	[BaseType (typeof (NSView))]
+	interface AUGenericView : AUCustomViewPersistentData {
+
+		[Export ("audioUnit")]
+		AudioUnit.AudioUnit AudioUnit { get; }
+
+		[Export ("showsExpertParameters")]
+		bool ShowsExpertParameters { get; set; }
+
+		[Export ("initWithAudioUnit:")]
+		IntPtr Constructor (AudioUnit.AudioUnit au);
+
+		[Export ("initWithAudioUnit:displayFlags:")]
+		IntPtr Constructor (AudioUnit.AudioUnit au, AUGenericViewDisplayFlags inFlags);
+	}
+
+	[NoiOS]
+	[Mac (10,13, onlyOn64: true)]
+	[BaseType (typeof (NSView))]
+	[DisableDefaultCtor]
+	interface AUPannerView {
+
+		[Export ("audioUnit")]
+		AudioUnit.AudioUnit AudioUnit { get; }
+
+		[Static]
+		[Export ("AUPannerViewWithAudioUnit:")]
+		AUPannerView Create (AudioUnit.AudioUnit au);
+	}
+
+	[NoiOS]
+	[Mac (10,13, onlyOn64: true)]
+	[BaseType (typeof (NSWindowController), Name = "CABTLEMIDIWindowController")]
+	interface CABtleMiDiWindowController {
+
+		[Export ("initWithWindow:")]
+		IntPtr Constructor ([NullAllowed] NSWindow window);
+	}
+
+	[NoiOS]
+	[Mac (10,13, onlyOn64: true)]
+	[BaseType (typeof (NSViewController))]
+	interface CAInterDeviceAudioViewController {
```

Comment:
base a default `init` make sense ?
or should not another base ctor/init be re-exposed ?

Reply:
Ok, found some sample code browsing it is using `[CAInterDeviceAudioViewController new];` so we should be fine